Sources/Sentry/SentryIOTrackingIntegration.m

```diff
@@ -0,0 +1,27 @@
+#import "SentryIOTrackingIntegration.h"
+#import "SentryNSDataSwizzling.h"
+#import "SentryOptions.h"
+
+@interface
+SentryIOTrackingIntegration ()
+
+@property (nonatomic, strong) SentryOptions *options;
+@end
+
+@implementation SentryIOTrackingIntegration
+
+- (void)installWithOptions:(SentryOptions *)options
+{
+    self.options = options;
+    if (options.enableSwizzling && options.enableAutoPerformanceTracking
```

Comment:
`l`: We have to update the features list of [`enableSwizzling` ](https://github.com/getsentry/sentry-cocoa/blob/229c663f6bdf225f8ef855f8355d48196a022cb8/Sources/Sentry/Public/SentryOptions.h#L260).